### DISCUSSION TITLE
Help needed to get Aspire Starter Template running on MacOS

### DISCUSSION THREAD

**Discussion by egrissom** [microsoft/aspire#4460] [Q&A] [answered]
Hi, I've had trouble getting a bare-bones Aspire Starter App to run on MacOS 14.5 (Sonoma). When I try to run AppHost, it fails when it tries to assign the services to an address.  I have tried disabling IPv6 on my Mac in case the issue was related, but even when I turn it off completely, it has the same result. The same happens if I run the ApiService. I can run the Web project, however.

Below are the errors I am seeing:

"Aspire.Hosting.Dcp.dcpctrl.ServiceReconciler: Error: could not start the proxy	{"ServiceName": {"name":"webfrontend_https"}, "Reconciliation": 7, "error": "cound not start the proxy for the service: listen tcp [fe80::1]:7088: bind: can't assign requested address"}
Aspire.Hosting.Dcp.dcpctrl.ServiceReconciler: Error: could not start the proxy	{"ServiceName": {"name":"webfrontend_http"}, "Reconciliation": 8, "error": "cound not start the proxy for the service: listen tcp [fe80::1]:5289: bind: can't assign requested address"}
fail: Aspire.Hosting.Dcp.dcpctrl.ServiceReconciler[0]
      could not start the proxy	{"ServiceName": {"name":"webfrontend_http"}, "Reconciliation": 8, "error": "cound not start the proxy for the service: listen tcp [fe80::1]:5289: bind: can't assign requested address"}"

**Comment by davidfowl** [comment]
cc @karolz-ms @danegsta

**Comment by danegsta** [comment, marked as answer]
@egrissom it looks like we're resolving fe80::1 instead of the more common ::1 address as `localhost` when trying to bind the service ports (but getting an error, because fe80::1 may not actually be associated with a network interface).

Can you double check your hosts file to see what addresses you have bound to `localhost`?

**Reply by egrissom** [reply]
Thanks for the response, checking it out I do see this: 
127.0.0.1	localhost
255.255.255.255	broadcasthost
::1             localhost 
fe80::1%lo0	localhost

**Reply by danegsta** [reply]
If you don't need that fe80::1 localhost entry for some other application/service, removing or commenting it out would be the quickest and easiest way to get you unblocked.

**Reply by egrissom** [reply]
Thank you! That did it. 

I use Local by Flywheel for Wordpress Dev and I am wondering if that us relying on it. I'll look into it more, but I this gets me going. Thanks, again.
